charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -146,7 +146,20 @@ data:
       {{- if .Values.global.networkNSLabels }}
         networkNSLabels: true
       {{- end }}
-
+      {{- if .Values.global.dagOnlyDeployment.enabled }}
+        dagDeploy:
+          enabled: {{ .Values.global.dagOnlyDeployment.enabled }}
+          images:
+            dagServer:
+              repository: {{ (splitList ":"  .Values.global.dagOnlyDeployment.image ) | first | quote }}
+              tag: {{ (splitList ":"  .Values.global.dagOnlyDeployment.image ) | last  }}
+        {{- if .Values.global.dagOnlyDeployment.enabled }}
+          securityContext: {{- .Values.global.dagOnlyDeployment.securityContext | toYaml | nindent 12 }}
```

Comment:
if openShift enabled should the default value be disabled?

Reply:
if you dont pass any values it will be passed as empty and we dont have any values behind the scenes